sshagentmux/base_agent_request.py

```diff
@@ -64,10 +64,10 @@ def handle(self):
         raise Exception('Unimplemented')
 
     def _key_digest(self, key_blob):
-        m = hashlib.md5()
+        m = hashlib.sha256()
```

Comment:
👍

Reply:
This change to to match `ssh-add` output. It will require existing users to update their database to sha256 hashes.